Description: Amend missing header changes as found in d337ea9c16edab254f1ce731d475a50a622886f9.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/winpr/include/winpr/thread.h
+++ b/winpr/include/winpr/thread.h
@@ -249,9 +249,7 @@
 
 	WINPR_API LPSTR* CommandLineToArgvA(LPCSTR lpCmdLine, int* pNumArgs);
 
-#if defined(WITH_DEBUG_THREADS)
 	WINPR_API VOID DumpThreadHandles(void);
-#endif
 
 #ifdef __cplusplus
 }
